packages/babel-types/test/converters.js

```diff
@@ -14,6 +14,7 @@ function generateCode(node) {
 describe("converters", function () {
   it("toIdentifier", function () {
     expect(t.toIdentifier("swag-lord")).toBe("swagLord");
+    expect(t.toIdentifier("ɵ2")).toBe("ɵ2");
```

Comment:
```suggestion
    expect(t.toIdentifier("ɵ2")).toBe("ɵ2");
    expect(t.toIdentifier("ℬ1").toBe("ℬ1");
```
Can add a non-BMP example.